torch/csrc/jit/passes/onnx/eval_peephole.cpp

```diff
@@ -119,12 +119,26 @@ static void fuseConvBatchNorm(Block* b, ValueToParamPairMap& valsToParamsMap) {
       valsToParamsMap.insert(
           {conv_W, std::make_pair(conv_W->debugName(), w_conv)});
       conv_W->inferTypeFrom(w_conv);
+      for (int j = 0; j < it->inputs().size(); j++) {
```

Comment:
Do we need this loop here? Isn't it known which input index is weight and which one is bias?

Reply:
Yes, we can know the index, but I'm not sure if the sequence is fixed for all of scenarios.